plugins/auth-keycloak/src/static/auth/keycloak/base/User.ts

```diff
@@ -0,0 +1,9 @@
+export interface keyCloakUser {
```

Comment:
Typo in interface :- 
```suggestion
export interface KeyCloakUser {
```